### PR TITLE
Enable/ disable revert using only a settings attribute

### PR DESCRIPTION
**Pull request by erikvw**
## Description
Some installations may not wish to allow users to revert original model instances via the history record. This PR adds a settings attribute ``SIMPLE_HISTORY_REVERT_DISABLED`` that if set to ``True`` removes the ``Revert`` button from the history form. 

## Motivation and Context
Makes it possible to globally disable the ability to revert a model instance via the historical record.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have run the `make format` command to format my code
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] I have added my name and/or github handle to `AUTHORS.rst`
- [x] I have added my change to `CHANGES.rst`
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by erikvw** [comment]
@rossmechanic, I've separated this functionality out of the larger PR (#631) as I think you had requested some time ago. The next step, if this is accepted, is to complete #631.

File: simple_history/templates/simple_history/submit_line.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <div class="submit-row">
-<input type="submit" value="{% trans 'Revert' %}" class="default" name="_save" {{ onclick_attrib }}/>
-{% if change_history %}<input type="submit" value="{% trans 'Change History' %}" class="default" name="_change_history" {{ onclick_attrib }}/>{% endif %}
-</div>
+  {% if not revert_disabled %}<input type="submit" value="{% trans 'Revert' %}" class="default" name="_save" {{ onclick_attrib }}/>{% endif %}
+  {% if change_history %}<input type="submit" value="{% trans 'Change History' %}" class="default" name="_change_history" {{ onclick_attrib }}/>{% endif %}
+  <a href="{{ history_url }}" class="closelink">{% trans 'Close' %}</a>
+</div>
```
Comment (gustavovaliati) [inline simple_history/templates/simple_history/submit_line.html]: Missing a new line.

File: simple_history/tests/tests/test_admin.py [hunk 2]
```diff
@@ -135,11 +141,13 @@ def test_history_list_custom_user_id_field(self):
 
         self.assertEqual(200, resp.status_code)
 
+    @tag("2")
```
Comment (gustavovaliati) [inline simple_history/tests/tests/test_admin.py]: Are these tags going to be permanent?

File: simple_history/templates/simple_history/object_history.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
 {% block content %}
   <div id="content-main">
 
-    <p>{% blocktrans %}Choose a date from the list below to revert to a previous version of this object.{% endblocktrans %}</p>
+    {% if revert_disabled %}<p>{% blocktrans %}Choose a date from the list below to revert to a previous version of this object.{% endblocktrans %}</p>{% endif %}
```
Comment (gustavovaliati) [inline simple_history/templates/simple_history/object_history.html]: Shouldn't this be `not revert_disabled`?

File: simple_history/tests/tests/test_admin.py (hunk 2 above)
Comment (erikvw) [reply simple_history/tests/tests/test_admin.py]: thanks! I'll remove them

File: simple_history/templates/simple_history/submit_line.html (hunk 1 above)
Comment (erikvw) [reply simple_history/templates/simple_history/submit_line.html]: thanks!

File: simple_history/templates/simple_history/object_history.html (hunk 3 above)
Comment (erikvw) [reply simple_history/templates/simple_history/object_history.html]: ah, yes. you are correct. thanks!

**Comment by erikvw** [comment]
thanks and agreed. using the admin permissions of the history objects,
which give you model level granularity, is the one way to go.

**Pull request merged**